demos/common/src/ie_config_helper.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "samples/ie_config_helper.hpp"
+
+void configureInferenceEngine(Core& ie,
+                              std::string& deviceString,
+                              std::string& deviceInfo,
+                              const std::string& lString,
+                              const std::string& cString,
+                              bool pc) {
+    for (char& ch : deviceString) {
+        ch = std::toupper(ch);
+    }
+
+    std::stringstream strBuffer;
```

Comment:
I don't think we really need the rest of the function:

It doesn't seem to be useful to print the version: we didn't even notice that some demos print just a ptr: https://github.com/openvinotoolkit/open_model_zoo/pull/1362 But version printing spams in terminal. Even if demos will keep printing the version, it's not about setting up ie.

There are no models left in omz that require CPU or GPU extensions.

`-pc` is usually for profiling a net. I would leave it to `benchmark_app`

Reply:
> There are no models left in omz that require CPU or GPU extensions.
> -pc is usually for profiling a net. I would leave it to benchmark_app

Then I can remove `-l`, `-c` and `-pc` from all demos, right?